### PR TITLE
ICDS Dashboard: Add loading indicator to mobile pages

### PR DESCRIPTION
**Pull request by czue**
can review all at once

### CODE REVIEW

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% load staticfiles %}
+<div class="position-fixed w-100 h-100vh d-flex justify-content-center align-items-center"
+     style="background: rgba(0,0,0,0.5); top: 0; z-index: 99;">
```
Comment (orangejenny) [inline custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html]: Is this going to interact properly with `.sideNavigation`?   https://github.com/dimagi/commcare-hq/blob/be1b3cc95752e2c7d347d526e91bf73bf2139054/custom/icds_reports/static/icds_reports/mobile/css/navigation-menu.less#L13

z index is an especially scary property to do as an inline style. Best practice is to use variables for z-index and to define them all in the same file - [here's where HQ does it](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less#L103-L112). It should be straightforward to add a `variables.less` file for the dashboard and move this into there.

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    <img src="{% static "icds_reports/mobile/images/loader.gif" %}" style="width: 125px; height: 125px">
```
Comment (orangejenny) [inline custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html]: For consistency with HQ, can you switch this to `<i class='fa fa-spin fa-spinner'></i>` and use font-size to resize it? We don't use loading images anymore - there's no harm in using them, this is purely a consistency comment. I also don't see the loader.gif file in the codebase.

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html (hunk 1 above)
Comment (czue) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html]: Nice catch and thanks for the feedback. Turns out this wasn't even needed so I just got rid of it.

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html (hunk 2 above)
Comment (czue) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html]: I swapped this in (and added the loader as well). I suspect we might get push back from product team but I'd rather do it this way too so let's see what they say.

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html (hunk 1 above)
Comment (orangejenny) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html]: > Turns out this wasn't even needed

Sweet, the fewer z indices the better!

**Review by orangejenny — APPROVED** [review]

**Pull request merged**